dotnet-trace/test.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+PROJNAME=dotnettrace
+FILENAME=dotnettrace.nettrace
+SPEEDSCOPENAME=dotnettrace.speedscope.json
+REPORTNAME=dotnettrace.nettrace.etlx
+
+
```

Comment:
Lets make the test install the tool dependency before using it.

```suggestion
dotnet tool uninstall -g dotnet-trace || true
dotnet tool install -g dotnet-trace

```

Reply:
You can use `dotnet tool update` instead.

If the tool isn't installed yet, it will install it.
If the tool is installed, it will update it to the latest version.